content/_assets/styles/fonts.scss

```diff
@@ -2,7 +2,12 @@
 // Fonts.scss
 // -----------------------------------------------------------------------------
 // Custom font declarations (if any) can be imported here
-$assets-dir: "/_assets/" !default;
+@media screen {
+  @import "assets-dir"
+}
+@media print {
+  @import "assets-dir-print"
+}
```

Comment:
@anderspollack I think it may be more clear to create two media specific SCSS modules: `content/_assets/styles/print.scss` and `content/_assets/styles/screen.scss` that each set `$assets-dir` and implement the `@function font($file)`, thoughts?